xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
+      "type": "number",
+      "description": "x-ray wavelength"
+    },
+    "bt_safN":{
+      "type": "string",
+      "description": "Safety Approval Number of beamtime"
+    },
+    "bt_experimenters":{
+      "type": "array",
+      "description": "list of experimenter names in [first, last,...] format"
+    },
+    "bt_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Beamtime object"
+    },
+    "sample_name":{
+      "type": "string",
+      "description": "name of sample being measured"
```

Comment:
You _can_, but you probably shouldn't for these meta-data reasons.  There is already a layer that provides nice iteration over / selecting out of (which is by header) so put really obvious distinctions at that that level.

headers should be granular enough that you do not do a whole lot of event-level filtering and coarse enough that you are not always merging headers in the first round or two of analysis.

ex doing a temperature ramps over a bunch of samples, doing one run per T ramp may make sense if once the sample is mounted and good probably going to stay good and the goal is to quickly get to some temperature v devrived_value curve and over the course of a beam time you will take many many such curves.  On the other hand, at any temperature the sample may explode or degrade, etc and over the course of a single beamtime you will be taking many readings at a single temperature and then at the end assembling / filtering all of them into a single T v derived_value curve, then one run per temperature point makes more sense.